packages/core/src/components/MenuToggle/styles.ts

```diff
@@ -26,4 +26,11 @@ export const styleSheetMenuToggle: StyleSheet = ({ unit, transition }) => ({
     display: 'flex',
     justifyContent: 'space-between',
   },
+
+  customToggle: {
+    border: 'none',
```

Comment:
There's a `resetButton` pattern I believe that can be used here.

Reply:
yeah, `pattern.resetButton` could help here